packages/crud/src/vaadin-crud.js

```diff
@@ -921,73 +910,97 @@ class Crud extends SlotMixin(ControllerMixin(ElementMixin(ThemableMixin(PolymerE
     form.addEventListener('input', this.__changeListener);
   }
 
-  /** @private */
-  __onSaveButtonChange(save, old) {
-    this.__setupSlottedButton(save, old, this.__saveBound);
-  }
-
-  /** @private */
-  __onDeleteButtonChange(deleteButton, old) {
-    this.__setupSlottedButton(deleteButton, old, this.__deleteBound);
+  /**
+   * @param {HTMLElement | undefined} form
+   * @param {string} theme
+   * @param {string | string[] | undefined} include
+   * @param {string | RegExp} exclude
+   * @private
+   */
+  __formPropsChanged(form, theme, include, exclude) {
+    if (form) {
+      form.include = include;
+      form.exclude = exclude;
+      form.setAttribute('theme', theme);
+    }
   }
 
-  /** @private */
-  __onCancelButtonChange(cancel, old) {
-    this.__setupSlottedButton(cancel, old, this.__cancelBound);
+  /**
+   * @param {HTMLElement} saveButton
+   * @param {HTMLElement} oldSaveButton
+   * @private
+   */
+  __onSaveButtonChange(saveButton, oldSaveButton) {
```

Comment:
```suggestion
  __onSaveButtonChanged(saveButton, oldSaveButton) {
```

Reply:
Done in https://github.com/vaadin/web-components/pull/3676.